### PR TITLE
update to use database configs

### PR DESCRIPTION
**Pull request by rgraber**

### CODE REVIEW

File: course_discovery/apps/course_metadata/migrations/0183_auto_20190614_1242.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+            model_name='tagcourseuuidsconfig',
+            name='course_uuids',
+            field=models.TextField(default=None, null=True, verbose_name='Course UUIDs'),
+        ),
+        migrations.AlterField(
+            model_name='tagcourseuuidsconfig',
+            name='tag',
+            field=models.TextField(default=None, null=True, verbose_name='Tag'),
+        ),
+    ]
```
Comment (dianakhuang) [inline course_discovery/apps/course_metadata/migrations/0183_auto_20190614_1242.py]: It's not a huge deal, but I think it's possible to merge these migrations into one, just for cleanliness sake.

File: course_discovery/apps/course_metadata/models.py [hunk 2]
```diff
@@ -2240,7 +2240,7 @@ class DrupalPublishUuidConfig(SingletonModel):
     """
     Configuration for data loaders used in the publish_uuids_to_drupal command.
     """
-    course_run_ids = models.TextField(default=None, null=False, blank=True, verbose_name=_('Course Run IDs'))
+    course_run_ids = models.TextField(default=None, null=True, blank=False, verbose_name=_('Course Run IDs'))
```
Comment (dianakhuang) [inline course_discovery/apps/course_metadata/models.py]: Why was this change made?

**Review by dianakhuang — APPROVED** [review]

File: course_discovery/apps/course_metadata/management/commands/add_tag_to_courses.py [hunk 3]
```diff
@@ -11,14 +11,27 @@ class Command(BaseCommand):
...
+                            )
 
     def handle(self, *args, **options):
-        self.add_tag_to_courses(options['tag'][0], options['courses'])
+        if options['args_from_database']:
+            optionsDict = self.get_args_from_database()
+            self.add_tag_to_courses(optionsDict['tag'], optionsDict['courses'].split())
+        else:
+            if options['tag'] is None or options['courses'] is None or options['courses'] == []:
+                raise CommandError(_('Missing required arguments'))
```
Comment (inventhouse) [inline course_discovery/apps/course_metadata/management/commands/add_tag_to_courses.py:26]: Might consider mutually-exclusive argument group so argparse can enforce required arguments when not getting args from the DB

File: course_discovery/apps/course_metadata/models.py (hunk 2 above)
Comment (rgraber) [reply course_discovery/apps/course_metadata/models.py]: Accident, and i thought i reverted that ><

**Review by inventhouse — APPROVED** [review]
Looks good; one suggestion but not sure it's worth the effort if this is tested and working. 👍

**Comment by rgraber** [comment]
git hell ><

**Pull request closed without merging**